spec/lib/order_spec.rb

```diff
@@ -125,6 +131,13 @@ def status=(new_value)
       expect(@order.old_status).to eq(0)
     end
 
+    it "check that order have paid status" do
```

Comment:
Should be "it checks". Not critical of course, but because it's opensource and people read it, we'd like to keep grammatically correct.